src/madetomeasure_api/db/core.clj

```diff
@@ -4,7 +4,7 @@
 
 (def db-spec
   {:subprotocol "postgresql"
-   :subname "//localhost/madetomeasure_api"
+   :subname "//localhost/madetomeasure_development?user=matthewkirk"
```

Comment:
Does YAML or environment vars make more sense?   I prefer YAML but I don't know what the world at large uses.